### ISSUE TITLE
Build `SubmissionPubHeader` "Submission" tab

### ISSUE THREAD

**Issue by idreyn** [pubpub/pubpub#1729]
Depends on #1728

This tab needs to dynamically update the `Pub` and `Submission` models when its fields are updated/blurred. We can give the `SubmissionPubHeader` these two props:

- `onUpdatePub(pub: Partial<Pub>) => unknown`
- `onUpdateSubmission(submission: Partial<Submission>) => unknown`

...and them plumb them down to the "Submission" tab. This tab has its own three sub-tabs:

- Title, Description & Abstract
- Contributors
- Pub Settings

Let's try to use the Blueprint `Tab(s)` here again and do a little work to see whether we can implement Deepak's bullets using `::after` in CSS. Otherwise drop.

### "Title, Description, and Abstract" tab

_[Figma link](https://www.figma.com/file/ERHvqJqCVBZ7dH2ZQBGlp8/SPUB-vision?node-id=297%3A1386)_

- `pub.title` (use Blueprint `FormGroup` + `InputGroup` to provide field name)
- `submission.abstract` (use `MinimalEditor` and the props in `WorkflowTextEditor` for reference, use `FormGroup` for field label)
- `pub.description` (use Blueprint `FormGroup` + `InputGroup` to provide field name and max-char label)

The Pub Content section is also at the bottom, we may want to workshop the copy. This section refers to the actual Pub body which users will see rendered below this component, but is out of scope for this task.

###  "Contributors" tab

_[Figma link](https://www.figma.com/file/ERHvqJqCVBZ7dH2ZQBGlp8/SPUB-vision?node-id=297%3A1462)_

This should only need to render some copy and a `PubAttributionEditor`, wired up to `onUpdatePub`, and you're done.

### "Pub Settings" tab

_[Figma link](https://www.figma.com/file/ERHvqJqCVBZ7dH2ZQBGlp8/SPUB-vision?node-id=297%3A1536)_

This has two sections, which consist of explanatory copy on the left and controls on the right

- "Header Background & Theme" has an Edit Pub Theme button that can basically reuse the code [here](https://github.com/pubpub/pubpub/blob/06cba5496549575051899ca643cedd6c12b94b79/client/containers/Pub/PubHeader/UtilityButtons.tsx#L37-L46), but using a BP button instead of a `SmallHeaderButton`
- "Default Download File" can use the `DownloadChooser`. This component exists in the `DashboardSettings` container — it should be moved to `components` and have the Dashboard-specific copy factored out of it and hoisted to its parent in the dashboard.
